README.md

```diff
@@ -10,7 +10,11 @@ Dolphin is a machine learning platform built on top of [Apache REEF](http://reef
 * [`dolphin-ps`](dolphin-ps/README.md): Parameter server module for asynchronous machine learning algorithms.
 
 ### How to build and run Dolphin
-1. Build Dolphin:
+
+1. Build REEF: check https://cwiki.apache.org/confluence/display/REEF/Compiling+REEF
+  Currently, Dolphin depends on REEF `0.14.0-incubating-SNAPSHOT`, which means you must build the current snapshot of REEF before building Dolphin. We will move to the release version `0.14.0-incubating` once it's out.
```

Comment:
Actually, the current snapshot version of REEF is `0.14.0-SNAPSHOT`; no more incubating!